src/useSubscription/models/channel.ts

```diff
@@ -43,6 +44,7 @@ export default class Channel<T extends Action = Action> {
   private readonly action: T
   private readonly args: ActionArguments<T>
   private readonly subscriptions: Map<number, Subscription<T>> = new Map()
+  private readonly scope = effectScope()
```

Comment:
I guess I expected scope to exist per subscription, not per channel. I don't see how this would cascade through children subscriptions if parent scope gets disposed?

Reply:
I think your misunderstanding is how subscriptions and channels are related. The channel executes the action and also cleans up after the action. Each subscription just gets the value of the action and helps inform the channel when the actions should be run.

Reply:
ah yes, I think I mixed up channel with manager